test/framework/framework.go

```diff
@@ -304,26 +304,18 @@ func (f *Framework) CreateOrUpdatePrometheusOperator(ctx context.Context, ns str
 		return nil, errors.Wrap(err, "wait for AlertmanagerConfig v1beta1 CRD")
 	}
 
-	// TODO(ArthurSens): The OperatorUpgrade tests won't pass because the operator v0.63.0 doesn't have the agent CRD.
-	// This check can be removed after the next release of the operator.
-	if createAgentCrd {
-		err = f.CreateOrUpdateCRDAndWaitUntilReady(ctx, monitoringv1alpha1.PrometheusAgentName, func(opts metav1.ListOptions) (runtime.Object, error) {
-			return f.MonClientV1alpha1.PrometheusAgents(v1.NamespaceAll).List(ctx, opts)
-		})
-		if err != nil {
-			return nil, errors.Wrap(err, "initialize PrometheusAgent v1alpha1 CRD")
-		}
+	err = f.CreateOrUpdateCRDAndWaitUntilReady(ctx, monitoringv1alpha1.PrometheusAgentName, func(opts metav1.ListOptions) (runtime.Object, error) {
+		return f.MonClientV1alpha1.PrometheusAgents(v1.NamespaceAll).List(ctx, opts)
+	})
+	if err != nil {
+		return nil, errors.Wrap(err, "initialize PrometheusAgent v1alpha1 CRD")
 	}
 
-	// TODO(xiu): The OperatorUpgrade tests won't pass because the operator v0.64.1 doesn't have the scrapeconfig CRD.
-	// This check can be removed after the next release of the operator.
-	if createScrapeConfigCrd {
-		err = f.CreateOrUpdateCRDAndWaitUntilReady(ctx, monitoringv1alpha1.ScrapeConfigName, func(opts metav1.ListOptions) (runtime.Object, error) {
-			return f.MonClientV1alpha1.PrometheusAgents(v1.NamespaceAll).List(ctx, opts)
-		})
-		if err != nil {
-			return nil, errors.Wrap(err, "initialize PrometheusAgent v1alpha1 CRD")
-		}
+	err = f.CreateOrUpdateCRDAndWaitUntilReady(ctx, monitoringv1alpha1.ScrapeConfigName, func(opts metav1.ListOptions) (runtime.Object, error) {
+		return f.MonClientV1alpha1.PrometheusAgents(v1.NamespaceAll).List(ctx, opts)
+	})
+	if err != nil {
+		return nil, errors.Wrap(err, "initialize PrometheusAgent v1alpha1 CRD")
```

Comment:
Looks like a mistake from the previous PR, but shouldn't this be installing ScrapeConfig here?